packages/utils/run-prettier.ts

```diff
@@ -19,7 +19,7 @@ export default function runPrettier(outputPath: string, source: string, cb?: Fun
 		try {
 			prettySource = prettier.format(source, {
 				filepath: outputPath,
-				parser: "babel",
+				parser: "babylon",
```

Comment:
Here they use `babel`: https://prettier.io/docs/en/api.html#prettierformatsource-options
Does it depend by the version that we have installed?

Reply:
You are right, `babylon` is deprecated. I'm still trying to figure out what is happening here, most of the time `babel` works but sometimes It gives me an error. I may have a dirty workspace in one of the computers I use... It'll be fixed before merging ;)